src/sqlfluff/core/rules/config_info.py

```diff
@@ -161,6 +161,13 @@
             "Defaults to ``False``."
         ),
     },
+    "prefer_quoted_keywords": {
+        "validation": [True, False],
+        "definition": (
+            "If ``True``, requires every keyword to be quoted. "
```

Comment:
To reduce ambiguity, can you change this to `every keyword used as an identifier`?